setup.py

```diff
@@ -460,15 +469,17 @@ def get_neuronxcc_version():
     if match:
         # Return the version string
         return match.group(1)
-    else:
-        raise RuntimeError("Could not find Neuron version in the output")
+
+    raise RuntimeError("Could not find neuron version in the output")
 
 
 def get_nvcc_cuda_version() -> Version:
     """Get the CUDA version from nvcc.
 
     Adapted from https://github.com/NVIDIA/apex/blob/8b7a1ff183741dd8f9b87e7bafd04cfde99cea28/setup.py
     """
+    from torch.utils.cpp_extension import CUDA_HOME
```

Comment:
`torch` import was removed at the top of this file

Reply:
As it is now, wouldn't the build fail if it tried to gett the NVCC version?

Reply:
It does not, although there are some issue related to aarch64, which I'll fix soon-ish